nni/retiarii/nn/pytorch/mutator.py

```diff
@@ -61,20 +68,14 @@ def __init__(self, nodes: List[Tuple[Node, str]], candidates: List[Any]):
     def mutate(self, model):
         chosen = self.choice(self.candidates)
         for node, argname in self.nodes:
+            chosen_value = node.operation.parameters[argname].access(chosen)
             target = model.get_node_by_name(node.name)
-            target.update_operation(target.operation.type, {**target.operation.parameters, argname: chosen})
+            target.update_operation(target.operation.type, {**target.operation.parameters, argname: chosen_value})
```

Comment:
seems you deal with indices of ValueChoice when ValueChoice is used as initialization parameters. what if the indices are used in `forward` function?

Reply:
You can find this case in the tests.

Reply:
yes, in test you uses valuechoice as a function's parameter. What if it is used naively? for example, `param = vc[0] + 0.1`? does it work?

Reply:
Test added. It works.

Reply:
haha, lihaile!